packages/fury-adapter-apib-serializer/test/adapter-test.js

```diff
@@ -35,7 +38,7 @@ describe('API Blueprint serializer adapter', () => {
         return done(loadErr);
       }
 
-      return serialize({ api }, (serializeErr, serialized) => {
+      return fury.serialize({ api }, (serializeErr, serialized) => {
```

Comment:
In this test we was testing the serializers adapter interface, I think it's better to test against Fury interface so we know the integration completely works.